inst/using-the-dccvalidator-1kD.Rmd

```diff
@@ -0,0 +1,125 @@
+---
+title: "Using the dccvalidator app in 1kD"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{using-the-dccvalidator-veoibd}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+We've built the dccvalidator tool to streamline the process of data validation and QA/QC. This application performs many of the routine data quality checks that were previously conducted by hand, with the hopes that it will help you, the data contributor, get your data checked, validated, and shared easily and quickly.
+
+## Application Requirements
+
+To use this application you must:
+
+1. Be logged in to Synapse in your browser
+2. Be a [Synapse certified user](https://docs.synapse.org/articles/accounts_certified_users_and_profile_validation.html){target="_blank"}
+3. Be a member of the [1kD team](https://www.synapse.org/#!Team:3382930){target="_blank"}
+
+Some portions of the app submit data to Synapse. This allows curators at Sage to troubleshoot issues if needed. No one outside the Sage VEO-IBD curation team will be able to download the data.
+
+## Data Sharing Agreement 
+
+In order to contribute data to the VEO-IBD Synapse project, you are required to sign the VEO-IBD Data Sharing Agreement.
+
+- If you discover you have released sensitive information in error, please follow the [process in Synapse to flag](https://docs.synapse.org/articles/contribute_and_access_controlled_use_data.html#flagging-inappropriate-data-use){target="_blank"} affected data. 
+
+- We are also interested in rapidly identifiying data that does not conform to our quality control standards. If you discover data is contaminated or sample identity is questionable due to a concordance analysis, for example, please reach out to the curation team - [VEOIBD_SageAdmin@synaspse.org](mailto:VEOIBD_SageAdmin@synapse.org){target="_blank"}.
```

Comment:
Change to an admin email for 1KD rather than VEOIBD?

Reply:
Yup will edit it accordingly today.